libs/shared/contentful/src/lib/contentful.client.spec.ts

```diff
@@ -48,6 +52,8 @@ describe('ContentfulClient', () => {
         graphqlSpaceId: faker.string.uuid(),
         graphqlEnvironment: faker.string.uuid(),
         firestoreCacheCollectionName: faker.string.uuid(),
+        memCacheTTL: DEFAULT_MEM_CACHE_TTL,
+        firestoreCacheTTL: DEFAULT_FIRESTORE_CACHE_TTL,
```

Comment:
This change doesn't seem necessary. If `memCacheTTL` isn't provided in the constructor, then `DEFAULT_MEM_CACHE_TTL` will be used by default by ContentfulClient. Similarly also applies to `DEFAULT_FIRESTORE_CACHE_TTL`